osbuild/util/chroot.py

```diff
@@ -37,5 +37,13 @@ def __enter__(self):
         return self
 
     def __exit__(self, exc_type, exc_value, tracebk):
+        failed_umounts = []
         for d in ["/proc", "/dev", "/sys"]:
-            subprocess.check_call(["/usr/bin/umount", self.root + d])
+            if subprocess.run(["/usr/bin/umount", self.root + d], check=False).returncode:
+                failed_umounts.append(d)
+        if failed_umounts:
+            print(f"Error unmounting paths from chroot: {failed_umounts}")
```

Comment:
I don't know.  I thought about it but landed on ignoring it and continuing.  Is it better to fail a build if one fails to unmount?

Reply:
I also don't know, but also I incline to just printing an error.

Reply:
Yeah, probably fince it, we could lazy umount for extra resilince but I guess I'm overthinking it now :) Let's keep as is.

Reply:
I would at least lazy unmount it -- since you don't know what'll be running in the chroot it's safer to assume they will be a bit slow to stop accessing things.